doc/nrf/working_with_nrf/nrf52/developing.rst

```diff
@@ -133,8 +133,13 @@ Consider using these features in your project to speed up the FOTA update proces
 FOTA in Bluetooth mesh
 ======================
 
-To perform FOTA update when working with the Bluetooth mesh protocol, use point-to-point Device Firmware Update (DFU) over Bluetooth Low Energy as described in `FOTA over Bluetooth Low Energy`_ above.
-See also :ref:`ug_bt_mesh_fota` for more details about FOTA in Bluetooth mesh.
+When performing a FOTA update when working with the Bluetooth mesh protocol, use one of the following DFU methods:
+
+* DFU over Bluetooth mesh using the Zephyr Bluetooth mesh DFU subsystem.
+  The specification that the Bluetooth mesh DFU subsystem is based on is not adopted yet, and therefore this feature should be used for experimental purposes only.
+* Point-to-point DFU over Bluetooth Low Energy as described in `FOTA over Bluetooth Low Energy`_ above.
+
+For more information about both methods, see :ref:`ug_bt_mesh_fota`.
```

Comment:
Isn't this basically the entire contents of the FOTA page now? Do we need the in-the-middle FOTA page anymore, can't we just keep the two DFU pages at top level and link them from here?

Reply:
I think we'd like to keep them like this for now.
Wille and I can discuss this post-release, how it is best to present things in the bt_mesh user guide. We should have a separate JIRA task for this.